vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -1,55 +1,50 @@
 <html layout:decorate="~{layouts/main_staff}">
-    <head>
-        <link rel="stylesheet"
-            href="https://unpkg.com/easymde/dist/easymde.min.css">
-        <script src="https://unpkg.com/easymde/dist/easymde.min.js"></script>
-        <link href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/css/select2.min.css" rel="stylesheet" />
-        <script src="//cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/js/select2.min.js"></script>
-		<script th:src="@{/resources/jquery/jquery-ui.min_custom_sortable.js}"></script>
-		<script src="//use.fontawesome.com/releases/v5.8.1/js/all.js"
-    	data-auto-replace-svg="nest"></script>
-        <style type="text/css">
-            .hova {
+<head>
+    <link rel="stylesheet" href="https://unpkg.com/easymde/dist/easymde.min.css">
+    <script src="https://unpkg.com/easymde/dist/easymde.min.js"></script>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/css/select2.min.css" rel="stylesheet" />
+    <script src="//cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/js/select2.min.js"></script>
+    <script th:src="@{/resources/jquery/jquery-ui.min_custom_sortable.js}"></script>
+    <style type="text/css">
+        .hova {
             background-color: #f2f2f2;
-            }
-            .EasyMDEContainer .CodeMirror-wrap{
+        }
+
+        .EasyMDEContainer .CodeMirror-wrap {
             overflow-y: scroll;
             height: 400px;
-            }
-            .fixed {
-		    position: fixed;
-		    top:0; left:0;
-		    width: 100%;
-		    z-index:100;
-     		}
-     		
-     		li.active>a{
-			background: solid var(--primary);
-			color: #fff;
-			font-size: 15px;
-			}
-			
-			a.nav-link.active.show {
-			border-color: transparent;
-			border-bottom: 4px solid var(--primary);
-			color: var(--primary) !important;
-			background: transparent;
-			}
-			
-			
-        </style>
-        <script th:inline="javascript">
-            //# sourceURL=click.js
+        }
+
+        .fixed {
+            position: fixed;
+            top: 0;
+            left: 0;
+            width: 100%;
+            z-index: 100;
+        }
+
+        .tooltip {
+            z-index: 0;
+        }
+    </style>
+
+    <script th:inline="javascript">
+    
+        	$(function () {
+            	$('.close').on('click', function () {
+                	$('.modal').hide();
+            	})
+        	})
```

Comment:
indentation

Reply:
addressed review comments where needed and made sure to address every review comment; please let me know if there is any comment I missed. I fixed the SpaceController one and almost all the indentation ones, and that why reopening the PR.

Reply:
indentation still